lib/ethers/transaction.ex

```diff
@@ -210,4 +261,13 @@ defmodule Ethers.Transaction do
 
   defp trim_leading(<<0, rest::binary>>), do: trim_leading(rest)
   defp trim_leading(<<bin::binary>>), do: bin
+
+  defp decode_key(tx, key) do
+    decoded_value = Map.get(tx, key) |> Ethers.Utils.hex_to_integer()
```

Comment:
Guess this may already be thought about. Would it be possible that key has value `nil`, will the `next_to_integer()` throw error in this case